lib/tidewave/mcp/tools/fs.ex

```diff
@@ -165,9 +161,25 @@ defmodule Tidewave.MCP.Tools.FS do
     not is_nil(connect_params["include_fs_tools"])
   end
 
-  def list_project_files(_args) do
-    with {:ok, files} <- GitLS.list_files() do
-      {:ok, Enum.join(files, "\n")}
+  def list_project_files(args) do
+    case args do
+      %{"glob_pattern" => glob_pattern} ->
+        git_ls_files(glob_pattern)
```

Comment:
Sorry, I think there is some confusion. If there is a a pattern, we don't call `git ls-files` at all.

Reply:
So you say we should use Path.wildcard? If we expect the built-in fs tools to only work with files in the project, it doesn't really make a difference, does it?

Reply:
I think we respect the current directory but we ignore the git filter.